lib/backpex/adapters/ash.ex

```diff
@@ -0,0 +1,114 @@
+defmodule Backpex.Adapters.Ash do
+  @config_schema [
+    resource: [
+      doc: "The `Ash.Resource` that will be used to perform CRUD operations.",
+      type: :atom,
+      required: true
+    ]
+  ]
+
+  @moduledoc """
+  The `Backpex.Adapter` to connect your `Backpex.LiveResource` to an `Ash.Resource`.
+
+  ## `adapter_config`
+
+  #{NimbleOptions.docs(@config_schema)}
+
+  > ### Work in progress {: .error}
+  >
+  > The `Backpex.Adapters.Ash` is not usable right now! It can barely list and show items. We will work on that, as we
+  > progress implementing the `Backpex.Adapter` pattern throughout the codebase.
```

Comment:
```suggestion
  > The `Backpex.Adapters.Ash` is currently not usable! It can barely list and show items. We will work on this as we continue to implement  the `Backpex.Adapter` pattern throughout the codebase.
```

The `Backpex.Adapters.Ash` is currently not usable! It can barely list and display items. We will work on this as we continue to implement the `Backpex.Adapter` pattern throughout the codebase.